modules/geo-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -86,6 +107,21 @@ export default class _MeshLayer extends SimpleMeshLayer {
     });
     return materialParser;
   }
+
+  calculateFeatureIdsPickingColors(attribute) {
+    const {featureIds} = this.props;
+    const value = new Uint8ClampedArray(featureIds.length * attribute.size);
+
+    for (let index = 0; index < featureIds.length; index++) {
+      const color = this.encodePickingColor(featureIds[index]);
```

Comment:
Reuse a pre-allocated array for better perf, see https://github.com/visgl/deck.gl/blob/8.4-release/modules/core/src/lib/layer.js#L513

Reply:
Done